GTG/gtk/browser/browser.py

```diff
@@ -1305,8 +1305,8 @@ def on_task_cursor_changed(self, selection=None):
         """
         # We unselect all in the closed task view
         # Only if something is selected in the active task list
-        self.donebutton.set_icon_name("gtg-task-done")
-        self.dismissbutton.set_icon_name("gtg-task-dismiss")
+        self.donebutton.set_icon_name(Gtk.STOCK_APPLY)
```

Comment:
Above you use set_stock_id for the button, down here you use set_icon_name. Could you use just single way how to set buttons? Is there a difference between those calls?